Doppler.Currency/Dtos/CurrencyDto.cs

```diff
@@ -12,5 +12,6 @@ public class CurrencyDto
         public decimal? BuyValue { get; set; }
         public string CurrencyName { get; set; }
         public string CurrencyCode { get; set; }
+        public bool NoPrice => SaleValue == 0 || BuyValue == 0;
```

Comment:
In my opinion, hiding this property in the JSON when it is false would be nice. 

Another option could be invertying the logic, something like this:

```
public bool cotizationAvailable => SaleValue != 0;
```

I have no a strong opinion on this.

Reply:
Ok,perfect, makes sence the change. I will change the property.